cli/src/plugins/javadoc/index.ts

```diff
@@ -201,15 +214,17 @@ function makeInheritedMethodList(args: MakeInheritedMethodListArgs) {
     md.list(
       "unordered",
       args.list.map((interfaceType) => {
-      return [
-        md.listItem([md.text(args.prefix + interfaceType.qualifiedTypeName),
-        md.list(
-          "unordered",
-          args.inheritedMethods[interfaceType.qualifiedTypeName].map((method) => {
-            return md.listItem(args.project.linkToEntity(interfaceType.qualifiedTypeName, method))
-          }))])
-      ]
-    }).flat(1))
+        return [
+          md.listItem(
+            [
+              md.text(args.prefix), args.project.linkToEntity(interfaceType.qualifiedTypeName), md.text(": "), md.paragraph(),
+              md.paragraph([args.inheritedMethods[interfaceType.qualifiedTypeName].map(
+                (method, index) => { return md.root([md.inlineCode(method + "()"), (index < args.inheritedMethods[interfaceType.qualifiedTypeName].length - 1 ? md.text(", ") : md.text(""))])}
```

Comment:
This doesn't seem right.

- If the method has parameters, it's not correct to call it `methodName()` (it would be `methodName(x, y, z)`)
- I think we're overusing monospace/inlineCode. Let's just do without then you can just use `.join(", ")` on an array of strings to create one string of text
- Unfortunately we don't have a "group" node in mdast. Technically root is reserved for a new document, so we should not be using it here.